mycroft/client/enclosure/weather.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2016 Mycroft AI, Inc.
+#
+# This file is part of Mycroft Core.
+#
+# Mycroft Core is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Mycroft Core is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Mycroft Core.  If not, see <http://www.gnu.org/licenses/>.
+
+
+from mycroft.util.log import getLogger
+
+__author__ = 'iward'
+
+LOGGER = getLogger(__name__)
+
+
+class EnclosureWeather:
+    """
+    Listens to enclosure commands to display indicators of the weather.
+
+    Performs the associated command on Arduino by writing on the Serial port.
+    """
+
+    def __init__(self, client, writer):
+        self.client = client
+        self.writer = writer
+        self.__init_events()
+
+    def __init_events(self):
+        self.client.on('enclosure.weather.display', self.display)
+
+    def display(self, event=None):
+        img_code, temp = 0, 0
+        if event and event.metadata:
+            img_code = event.metadata.get("img_code", img_code)
+            temp = event.metadata.get("temp", temp)
```

Comment:
Is that correct to implement a default value of 0 for img and 0 for temp? I mean, if we send 0 and 0 to the enclosure arduino code, what will happen? Taking a step back, it seems we should not event send anything to the unit if we don't have valid values for img and temp. Don't you agree?